index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="./style.css" />
+  </head>
+
+  <body>
+    <p id="para-1">Hello World we are software developers!</p>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <main id="main">
+      <title id="title">Donald Trump</title>
+      <div id="img-div">
+        <img
+          id="image"
+          src="https://i.guim.co.uk/img/media/70d12c1d77662696f75a21d04ec1bb5e8a702390/60_37_4451_2672/master/4451.jpg?width=465&quality=45&auto=format&fit=max&dpr=2&s=209e10f34772bc8b4e9a58fa8227dd5a"
+          alt="Donald Trump waving to crowd"
+          class="responsive"
+        />
+        <p id="img-caption">Donald Trump waving to the crowd.</p>
+      </div>
+      <h2 id="tribute-info">Donald Trump's Information</h2>
+      <p>
+        Donald John Trump (born June 14, 1946) is an American politician, media
+        personality, and businessman who served as the 45th president of the
+        United States from 2017 to 2021. Born and raised in Queens, New York
+        City, Trump graduated from the Wharton School of the University of
+        Pennsylvania with a bachelor's degree in 1968. He became president of
+        his father Fred Trump's real estate business in 1971 and renamed it The
+        Trump Organization. Trump expanded the company's operations to building
+        and renovating skyscrapers, hotels, casinos, and golf courses. He later
+        started various side ventures, mostly by licensing his name. From 2004
+        to 2015, he co-produced and hosted the reality television series The
+        Apprentice. Trump and his businesses have been involved in more than
+        4,000 state and federal legal actions, including six bankruptcies.
+      </p>
+      <a
+        id="tribute-link"
+        href="https://en.wikipedia.org/wiki/Donald_Trump"
+        target="_blank"
+        >For more information about Donald John Trump</a
+      >
```

Comment:
It is good practice to separate parts of the website into sections. It would be a good idea to start implementing that into our website and try to stop only using divs. Keep up the good work! O(∩_∩)O